apps/whale-api/src/module.api/address.controller.e2e.ts

```diff
@@ -9,6 +9,9 @@ import { Testing } from '@defichain/jellyfish-testing'
 import { ForbiddenException } from '@nestjs/common'
 import BigNumber from 'bignumber.js'
 import { RegTestFoundationKeys } from '@defichain/jellyfish-network'
+import { FutureSwap } from '@defichain/jellyfish-api-core/src/category/account'
+
+/* eslint-disable no-lone-blocks */
```

Comment:
I think this can be removed.

Reply:
we have restriction
https://github.com/JellyfishSDK/jellyfish/blob/0d8264f97b6297c667681414364e52b0b26b0056/.eslintrc#L62

Reply:
hmm.. that part is not under coverage

Reply:
Okay, my bad. You are right.